apps/webstore/src/app/pages/products/product-pages/family-tree/family-tree.component.html

```diff
@@ -175,7 +175,7 @@
 
             <div class="option">
                 <button class="tc-button-100" (click)="openTemplateModal()" data-cy="family-tree-template-button">
-                    <span i18n="@@use-template">Use template</span>
+                    <span i18n="@@select-a-template">Select a template</span>
```

Comment:
this requires changes to the localization files. If you wish to change text on the page without a danish supervision, then search for the text (in this case "Use template") in the messages.xlf file and change it there. Avoid changing the i18n id since that will create a new translation that is then not present on the danish one.

If you want to add the text change anyways, then run: 
`nx extract-i18n webstore`

And update the message.xlf and message.da.xlf files with the new translations.